chapters/6-analysis.tex

```diff
@@ -430,52 +457,55 @@ \subsection{Visualizing data}
 as does \textit{Data Visualization: A Practical Introduction}.\sidenote{
 	\url{http://socvis.co}}
 Graphics tools in Stata are highly customizable.
-There is a fair amount of learning curve associated with extremely-fine-grained adjustment,
+There is a fair amount of learning curve associated with
+extremely-fine-grained adjustment,
 but it is well worth reviewing the graphics manual.\sidenote{\url{
 		https://www.stata.com/manuals/g.pdf}}
-For an easier way around it, Gray Kimbrough's 
+For an easier way around it, Gray Kimbrough's
 \textit{Uncluttered Stata Graphs}
-code is an excellent default replacement for Stata graphics that is easy to install.\sidenote{
+code is an excellent default replacement for Stata graphics
+that is easy to install.\sidenote{
```

Comment:
There are a million user-written schemes out there, and which one to use is very subjective. I think we should either list more than one, or just mention their existence without pointing to any single one in particular.

Reply:
put on wiki